.gitignore

```diff
@@ -1,3 +1,4 @@
+/node_modules
 package-lock.json
-node_modules
-/*Agro_easy
+/New folder
+.gitconfig
```

Comment:
remove `/New Folder` and `.gitconfig` from the `. gitignore` they have nothing to do with this repo

Reply:
done